src/screens/ElderlyEmotionHistory.tsx

```diff
@@ -0,0 +1,83 @@
+import { View, useToast } from 'native-base';
+import React, { useState, useEffect } from 'react';
+import { useWindowDimensions } from 'react-native';
+import { ContributionGraph } from 'react-native-chart-kit';
+import EmotionalTable from '../components/molecules/EmotionalTable';
+import { EmotionalHistory } from './../dto/modules/emotionRecord';
+import {
+  EmotionalHistoryFrequency,
+  getEmotionAsHeatmapFrequency
+} from './../utils/caretaker/emotionHeatmap';
+import moment from 'moment';
+
+const ElderlyEmotionHistory = () => {
+  const { width, height } = useWindowDimensions();
+  const [hist, setHist] = useState<EmotionalHistory[]>([]);
+
+  const toast = useToast();
+  function handleDayPress(val: EmotionalHistoryFrequency) {
+    toast.show({ title: JSON.stringify(val) });
+  }
+
+  useEffect(() => {
+    const data: EmotionalHistory[] = [
```

Comment:
This is the old version of mine before connecting to back-end, can you change it to the newest version?